.github/workflows/release-compose.yml

```
@@ -0,0 +1,76 @@
+          response=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/${{ github.repository }}/releases/tags/${{ github.ref_name }}")
+          if echo "$response" | grep -q "id"; then
+            upload_url=$(echo "$response" | jq -r '.upload_url')
+            echo "release_exists=true" >> $GITHUB_ENV
+            echo "upload_url=$upload_url" >> $GITHUB_OUTPUT
+          else
+            echo "release_exists=false" >> $GITHUB_ENV
+          fi
+
+      - name: Get Previous Tag
+        id: get_previous_tag
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:6: Double quote to prevent globbing and word splitting

(shellcheck)